### PR TITLE
adding google-style docstring to print_version

### PR DESCRIPTION
**Pull request by komish**
# Description

Adding additional google-style docstrings as requested in #532. Thanks for participating in Hacktoberfest 2019

## Status
READY

## Type of change
- Documentation (fix or adds documentation)

### CODE REVIEW

File: opsdroid/cli/version.py [hunk 1]
```diff
@@ -8,7 +8,19 @@
 
 
 def print_version(ctx, param, value):
-    """[Deprecated] Print out the version of opsdroid that is installed."""
+    """[Deprecated] Print out the version of opsdroid that is installed.
+
+    Args:
+        ctx (:obj:`click.Context`): The current click cli context
```
Comment (jacobtomlinson) [inline opsdroid/cli/version.py]:
```suggestion
        ctx (:obj:`click.Context`): The current click cli context.
```

Comment (komish) [reply opsdroid/cli/version.py]: Should be resolved in the latest change.

**Comment by komish** [comment]
> In future if you make sure the allow edits from maintainers checkbox is checked we could make these minor tweaks ourselves without having to come back to you for it.

Sorry about that! Didn't even notice I unticked the box. 

I've pushed the changed and attempted a rebase with new upstream commits. Please let me know if I've done it incorrectly and need to make adjustments.

**Comment by FabioRosado** [comment]
Thanks for fixing this issue. Since you fixed the little thing that Jacob suggested, I'm going to merge this PR now.

We love to thank contributors by sending them [opsdroid stickers](https://medium.com/opsdroid/contributor-sticker-packs-738058ceda59)! Fill in the [claim form](https://forms.gle/iQZV9DjkaUiSaDXK6) to get yours!

**Pull request merged**